config/abilities/provider_any.rb

```diff
@@ -98,8 +101,7 @@
 
     can :manage, AccessToken, owner_id: user.id
 
-    if can?(:manage, :partners) && can?(:manage, account)
-      can :export, :data
-    end
+    can :export, :data if can?(:manage, :partners) && can?(:manage, account)
   end
+  # rubocop:enable Metrics/BlockLength
```

Comment:
IMO it's better to approve issues in code climate before merge than permanently disabling checks in source. In this way by a later refactoring, one may decide to actually fix these.

Reply:
Or, if we don't care about some rules, we can as well just disable them.

Reply:
I'm used to have the cops disabled directly in the code. The reasoning is that it makes clear for any developer that the file has some style (or worse) issue that was caught but not addressed in the past. It also puts the code as the source of truth instead of a 3rd party application. (I actually had troubles with it in the past once a Company stopped using CodeClimate in favour of another application).

Reply:
In general I approve with @akostadinov but this one particularly will never be fixed 🤷🏻‍♂️ `Metrics/BlockLength`.